spyder/plugins/completion/languageserver/plugin.py

```diff
@@ -595,15 +597,22 @@ def generate_python_config(self):
         }
```

Comment:
Also maybe add a test here to check the generated config contains expected values?